daily_snapshot_terraform/modules/daily_snapshot/service/upload_snapshot.sh

```diff
@@ -57,7 +57,8 @@ echo "Chain: $CHAIN_NAME"
 echo "Snapshot: $NEWEST_SNAPSHOT"
 
 # spawn a task in the background to periodically write Prometheus metrics to a file
-write_metrics &
+# Disabled to reduce logging noise. Re-enable if Forest starts having network troubles again.
+# write_metrics &
```

Comment:
They may be helpful. The noise comes from the fact that we have the debug option set with `x` that logs every single operation. I'd rather we remove this and not metrics.

Reply:
Hmm, right. Let's take a step back and re-think how we track the different logs.

Reply:
related https://github.com/ChainSafe/forest-iac/issues/197